src/server/index.js

```diff
@@ -73,8 +73,8 @@ Promise.all([fetchParameters(), fetchVariables()])
       countryPackageName: country_package_name,
       countryPackageVersion: country_package_version,
       currency,
-      parameters: withSearchIndex('description', parameters),
-      variables: withSearchIndex('label', variables),
+      parameters: withNormalizedDescription('description', parameters),
+      variables: withNormalizedDescription('label', variables),
```

Comment:
This call doesn't seem to be legit, since `withNormalizedDescription` only takes a single parameter.